### PR TITLE
Show cluster in place details

### PR DESCRIPTION
**Pull request by mutaphysis**
Based on #194

### CODE REVIEW

File: src/components/NodeToolbar/AccessibilitySection/PlaceAccessibilitySection.js [hunk 1]
```diff
@@ -47,7 +48,7 @@ export default function PlaceAccessibilitySection(props: Props) {
   const descriptionElement = description ? <Description>{description}</Description> : null;
 
   return (
-    <StyledFrame>
+    <StyledFrame noseOffsetX={cluster ? 67 : undefined}>
```
Comment (opyh) [inline src/components/NodeToolbar/AccessibilitySection/PlaceAccessibilitySection.js]:
```suggestion
    <StyledFrame arrowOffsetX={cluster ? 67 : undefined}>
```

🐽

File: src/components/NodeToolbar/AccessibilitySection/StyledFrame.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ const StyledFrame = styled.div`
     position: absolute;
     content: ' ';
     top: -8px;
-    left: 10px;
+    left: ${props => props.noseOffsetX || 10}px;
```
Comment (opyh) [inline src/components/NodeToolbar/AccessibilitySection/StyledFrame.js]:
```suggestion
    left: ${props => props.arrowOffsetX || 10}px;
```

**Comment by opyh** [comment]
Problems I found so far when trying this out:

When clicking on a cluster, I get an exception
screenshot 2019-02-01 at 19 10 01

Red clusters should be a horizontal/vertical square, not a diamond:
screenshot 2019-02-01 at 19 08 46

Bigger clusters should be rendered bigger than smaller clusters, as before:
screenshot 2019-02-01 at 19 08 37

Text color should be white:
screenshot 2019-02-01 at 19 08 31

**Comment by mutaphysis** [comment]
I will check the stack overflow, but I have a feeling that’s also the same problem. It happened during hot reloads for me as well.

**Comment by opyh** [comment]
Fixed the crash.

```
if (this.state.zoomedToFeatureId !== targetMapState.zoomedToFeatureId) {
   this.setState({
     zoomedToFeatureId: targetMapState.zoomedToFeatureId,
   });
}
```

sets a state within a `componentDidUpdate` callback that happens after `setState`, leading to an infinite loop.

I've commented this out for now. It's okay for now if the UI zooms again when clicking on a cluster twice IMO.

**Pull request merged**